yigitcanTask/force-app/main/default/pages/accountDetails1.page

```diff
@@ -0,0 +1,18 @@
+<apex:page standardController="Account">
```

Comment:
if this page is not used - please remove it and do not include into pull request